src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -263,4 +268,34 @@ export const ANALYTICS_EVENTS = {
     [HELP_TEXT_CLICKED_EVENT]:
       'Disability - Form 4142 - Private Medical Records Release: What does this mean',
   },
+  openedPtsdTypeHelp: {
+    event: 'form-help-text-clicked',
+    [HELP_TEXT_CLICKED_EVENT]:
+      'Disability - PTSD Intro - Which should I choose',
+  },
+  openedPtsd781WalkthroughChoiceHelp: {
+    event: 'form-help-text-clicked',
+    [HELP_TEXT_CLICKED_EVENT]:
+      'Disability - Form 21-0781 - Which should I choose',
+  },
+  openedPtsd781aWalkthroughChoiceHelp: {
+    event: 'form-help-text-clicked',
+    [HELP_TEXT_CLICKED_EVENT]:
+      'Disability - Form 21-0781a - Which should I choose',
+  },
+  openedPtsd781IncidentDateHelp: {
+    event: 'form-help-text-clicked',
+    [HELP_TEXT_CLICKED_EVENT]:
+      'Disability - Form 21-0781 - What if I can’t remember the date',
+  },
+  openedPtsd781aIncidentDateHelp: {
+    event: 'form-help-text-clicked',
+    [HELP_TEXT_CLICKED_EVENT]:
+      'Disability - Form 21-0781a - What if I can’t remember the date',
+  },
+  openedPtsd781aOtherSourcesHelp: {
```

Comment:
This seems to be a duplicate of `openedPtsd781aWalkthroughChoiceHelp` so need some guidance on new text.